vnstatd.te

```diff
@@ -34,20 +38,21 @@ allow vnstatd_t self:process signal;
 allow vnstatd_t self:fifo_file rw_fifo_file_perms;
 allow vnstatd_t self:unix_stream_socket { accept listen };
 
+manage_dirs_pattern(vnstatd_t, vnstatd_runtime_t, vnstatd_runtime_t)
+manage_files_pattern(vnstatd_t, vnstatd_runtime_t, vnstatd_runtime_t)
+files_pid_filetrans(vnstatd_t, vnstatd_runtime_t, { dir file })
+
 manage_dirs_pattern(vnstatd_t, vnstatd_var_lib_t, vnstatd_var_lib_t)
 manage_files_pattern(vnstatd_t, vnstatd_var_lib_t, vnstatd_var_lib_t)
-files_var_lib_filetrans(vnstatd_t, vnstatd_var_lib_t, { dir file })
```

Comment:
Unfortunately there are many things I'd like higher-privilege processes to do on behalf of lesser/un-privileged domains, but it doesn't always happen.